components/03-modules/slider/_slider.scss

```diff
@@ -0,0 +1,186 @@
+        padding: 0;
+        min-width: 50%;
+        padding-right: $slider__item-padding-right;
+        border: 0;
+
+        @include mq($screen-m) {
+            min-width: 33%;
+        }
+
+        @include mq($screen-l) {
+            min-width: 25%;
+        }
+
+        .product__img-box {
+            min-height: unset;
+        }
+
+        .product__type {
+            padding: 0;
+        }
+    }
+
+    &__link {
+        text-decoration: none;
+
+        &:hover {
+            text-decoration: none;
+        }
+    }
+
+    &__image {
+        width: 100%;
+        margin: $slider__image-margin;
+    }
+
+    &__item-handler {
+        display: flex;
+        flex-wrap: wrap;
+
+        @include mq($screen-m) {
+            flex-wrap: nowrap;
+            justify-content: space-between;
+        }
+    }
+
+    &__name {
+        flex-basis: 100%;
+        font-weight: $slider__name-font-weight;
+
+        @include mq($screen-m) {
+            flex-basis: auto;
+            margin-right: $slider__name-margin-right;
+            margin-bottom: 0;
+        }
+
+        @include mq($screen-l) {
+            font-size: $slider__name-font-size;
+        }
+    }
+
+    &__price {
+        display: flex;
+        flex-wrap: wrap;
+        align-items: center;
+        color: $slider__price-color;
+        font-weight: $slider__price-font-weight;
+
+        @include mq($screen-m) {
+            flex-flow: column;
+            align-items: flex-end;
+            min-width: fit-content;
+        }
+    }
+
+    &__price-old {
+        display: block;
+        margin-right: $slider__price-old-margin;
+        color: $slider__price-old-color;
+        font-weight: $slider__price-old-font-weight;
+        font-size: $slider__price-old-font-size;
+        text-decoration: line-through;
+
+        @include mq($screen-m) {
+            margin-right: 0;
+        }
+    }
+
+    &__price-new {
+        color: $slider__price-new-color;
+        font-size: $slider__price-new-font-size;
+        font-weight: $slider__price-new-font-wieght;
+    }
+
+    &__badge {
+        position: absolute;
+        top: $slider__badge-left;
+        left: 0;
+    }
+
+    &__navigation {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        margin-top: $slider__navigation-margin;
+
+        @include mq($screen-m) {
+            display: none;
+        }
+    }
+
+    &__navigation-icon {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: $slider__navigation-icon-spacing;
+        width: $slider__navigation-icon-spacing;
+
+        &--left {
+            margin-right: $slider__navigation-icon-margin;
+            left: 0;
+        }
+
+        &--right {
+            margin-left: $slider__navigation-icon-margin;
+            right: 0;
+        }
+    }
+
+    &__icon {
+        width: $slider__icon-spacing;
+        height: $slider__icon-spacing;
```

Comment:
I think this `$slider__icon-spacing` variable should be named `$slider__icon-size` / `sizing` cause you're using it only for a width and height properties...

Reply:
renamed